nixos/modules/services/misc/gitea.nix

```diff
@@ -357,12 +384,22 @@ in
     };
 
     systemd.tmpfiles.rules = [
```

Comment:
@kolaente just curious why `gitea` does not provision any of its required sub directories itself. Do you have any insight on that?

Reply:
You mean the `conf`, `custom` and `log` directories? IIRC it does that if you install it through the web wizard but probably not when installing it the way it is installed in nix.

(I'll ask the other maintainers if they know more about this)